CSS/style.css

```diff
@@ -0,0 +1,216 @@
+
+body {
+  font-family: Lato, cocogoose, Tahoma, sans-serif;
+}
+
+header {
+  height: 65px;
+  position: fixed;
+}
+
+.only-desktop {
+  display: none;
+}
+
+.only-mobile {
+  display: block;
+}
+
+.hamburger-icon img {
+  width: 40px;
+  padding-left: 15px;
+  padding-top: 15px;
+}
+
+#headline-section {
+  background: url("../Images/waraMobile.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 100px 12px 20px 15px;
+}
+
+.headline-title-text p {
+  color: #ec5242;
+  font-size: 23px;
+  font-weight: 400;
+  margin: 0;
+}
+
+.headline-title-text h2 {
+  background: -webkit-linear-gradient(#d52b1a, #ec5242);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  font-size: 35px;
+  font-weight: 900;
+  margin: 5px 0;
+}
+
+.headline-box-text {
+  border: 2px solid white;
+  margin-top: 30px;
+}
+
+.headline-box-text p {
+  padding: 0 25px;
+  color: darkslategray;
+  line-height: 23px;
+}
+
+.headline-bottom-text {
+  padding-left: 15px;
+}
+
+.headline-bottom-text h3 {
+  font-size: 25px;
+  font-weight: 900;
+  color: darkslategray;
+  margin-bottom: 0;
+}
+
+.headline-bottom-text p {
+  color: darkslategray;
+}
+
+#main-program {
+  background: #282a30;
+}
+
+.main-program-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.main-program-container button {
+  background: #ec5242;
+  padding: 20px;
+  margin: 40px;
+  color: #fff;
+  font-size: 16px;
+  border: none;
+}
+
+#main-program h3 {
+  color: #e9e3e3;
+  text-align: center;
+  padding-top: 20px;
+  font-size: 25px;
+  font-weight: 400;
+  margin-bottom: 0;
+}
+
+.main-line {
+  background-color: #ec5242;
+  height: 2px;
+  width: 60px;
+  margin: 15px auto 50px auto;
+}
+
+.program-catagories {
+  display: flex;
+  justify-content: space-evenly;
+  background-color: #e5dbdb3d;
```

Comment:
Kindly follow these design guidelines: 🙂 
  - Colors.
  - Typography: font face, size, and weight.
  -  Layout: composition and space between elements.
  

Here is a screenshot of the colors and typography used 👍 
![Screenshot (307)](https://user-images.githubusercontent.com/66112760/141968910-da585578-fbc7-4ebf-a533-5e2d30ad896b.png)